pandas/tests/test_arithmetic.py

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+# Arithmetc tests for DataFrame/Series/Index/Array classes that should
+# behave identically.
+from datetime import timedelta
+
+import pytest
+import numpy as np
+
+import pandas as pd
+import pandas.util.testing as tm
+
+from pandas import Timedelta
+
+
+def assert_equal(left, right, **kwargs):
+    """
+    Wrapper for tm.assert_*_equal to dispatch to the appropriate test function.
+
+    Parameters
+    ----------
+    left : Index, Series, or DataFrame
+    right : Index, Series, or DataFrame
+    **kwargs
+    """
+    if isinstance(left, pd.Index):
+        tm.assert_index_equal(left, right, **kwargs)
+    elif isinstance(left, pd.Series):
+        tm.assert_series_equal(left, right, **kwargs)
+    elif isinstance(left, pd.DataFrame):
+        tm.assert_frame_equal(left, right, **kwargs)
+    else:
+        raise NotImplementedError(type(left))
+
+
+def box_expected(expected, box_cls):
```

Comment:
maybe put in conftest.py, these are test helper functions that we don't necessarily want to publicly expose

Reply:
alternatively, we should put this into _test_decorators (maybe rename though)